examples/python/tests/getting_started/test_first_script.py

```diff
@@ -23,3 +23,6 @@ def test_eight_components():
     assert value == "Received!"
 
     driver.quit()
+
+
+test_eight_components()
```

Comment:
This is not needed because we use `pytest` to run the examples. So ideally the instructions should mention `pytest`.